spring-cloud-open-service-broker-autoconfigure/src/main/java/org/springframework/cloud/servicebroker/autoconfigure/web/ApiVersionWebMvcAutoConfiguration.java

```diff
@@ -49,7 +49,7 @@
 @Configuration
 @ConditionalOnWebApplication(type = ConditionalOnWebApplication.Type.SERVLET)
 @ConditionalOnBean({ ServiceInstanceService.class })
-@ConditionalOnProperty(prefix = "spring.cloud.openservicebroker", name = "apiVersionCheckEnabled", havingValue = "true", matchIfMissing = true)
+@ConditionalOnProperty(prefix = "spring.cloud.openservicebroker", name = "api-version-check-enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
@AlbertoImpl This looks good, but the `@ConditionalOnProperty(prefix = "spring.cloud.openservicebroker", name = "apiVersion")` just below should be changed to use the canonical form also.